core/domain/image_validation_services_test.py

```diff
@@ -129,3 +129,18 @@ def test_image_validation_checks(self) -> None:
             feconf.ENTITY_TYPE_EXPLORATION,
             'Expected a filename ending in .png'
         )
+        base64_encoded_string = 'SGVsbG8gV29ybGQh'
+        self._assert_image_validation_error(
+            base64_encoded_string,
+            'image.svg',
+            feconf.ENTITY_TYPE_EXPLORATION,
+            'Image not recognized'
+        )
```

Comment:
Isn't this already tested in lines:
https://github.com/oppia/oppia/blob/5deb24c619f3c99360e89babb2874e89c83110cb/core/domain/image_validation_services_test.py#L101-L106

Reply:
https://github.com/oppia/oppia/blob/3ad45bda3ac4c8155d4c92eb53bf78b96988128c/core/domain/image_validation_services.py#L58
https://github.com/oppia/oppia/blob/3ad45bda3ac4c8155d4c92eb53bf78b96988128c/core/domain/image_validation_services.py#L59
missing coverage line 59
To execute the line 59, line 58 must be true.
'SGVsbG8gV29ybGQh' is a string which is _base64_encoded.
and b'not an image' is not _base64_encoded and is bytes.